io/disk/sysfs_att.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Maram Srimannarayana Murthy <Maram.Srimannarayana.Murthy@ibm.com>
+
+"""
+Check all sysfs queue attributes by reading or writing different values
+"""
+
+import os
+from avocado import Test
+from avocado.utils import genio
+
+
+class Sysfs_Queue(Test):
+
+    """
+    Checks all Sysfs_Queue attributes
+    """
+
+    def setUp(self):
+        """
+        Reading required arguments from yaml file
+        Setting up file paths
+        """
+        self.queue_attr_disk = self.params.get("disk", default=None)
+        self.log.info("disk: %s", self.queue_attr_disk)
+        self.queue_attr_disk_dir = os.path.join("/sys/block",
+                                                self.queue_attr_disk, "queue")
+        self.attribute_name = self.params.get("attribute_name",
```

Comment:
instead attribute, I feel parameter fits good. re-think on it.

Reply:
Update variable name as per your comment